src/logs.ts

```diff
@@ -1,13 +1,4 @@
 export const enum LogLevel {
-  Trace = 'trace',
-  Debug = 'debug',
-  Info = 'info',
-  Warn = 'warn',
-  Error = 'error',
-  Fatal = 'fatal',
-}
-
-export const enum LogLevelNum {
```

Comment:
Question: why do we need to have both verbose and numeric levels?

Reply:
Numeric is great for comparison:

```
if (log.level > LogLevelNum.info) {
  // ... send to server to store for later debugging, etc
}
```

Pretty `LogLevel` is great for reporting and human consumption.